libs/tools/export/vault-export/vault-export-core/src/services/individual-vault-export.service.ts

```diff
@@ -169,9 +171,13 @@ export class IndividualVaultExportService
       }),
     );
 
+    const restrictedTypes = await this.getRestrictedTypes();
+
     promises.push(
       this.cipherService.getAllDecrypted(activeUserId).then((ciphers) => {
         decCiphers = ciphers.filter((f) => f.deletedDate == null);
+
+        decCiphers = decCiphers.filter((c) => !restrictedTypes.includes(c.type));
```

Comment:
⛏️ This is iterating through the list of ciphers twice, combining them into a single `filter` will be more efficient

```suggestion
        decCiphers = ciphers.filter((f) => f.deletedDate == null && !restrictedTypes.includes(c.type));
```